app/src/layouts/organizationsControl/organizationsPopover/organizationsPopover.jsx

```diff
@@ -38,22 +42,74 @@ export const messages = defineMessages({
     id: 'OrganizationsControl.assignmentsList',
     defaultMessage: 'Assignments list',
   },
+  search: {
+    id: 'OrganizationsControl.search',
+    defaultMessage: 'Search',
+  },
+  noResultsFound: {
+    id: 'OrganizationsControl.noResultsFound',
+    defaultMessage: 'No results found',
+  },
 });
 
-export const OrganizationsPopover = ({ closePopover, closeSidebar }) => {
+export const OrganizationsPopover = ({ closePopover, closeSidebar, variant }) => {
   const { formatMessage } = useIntl();
   const availableProjects = useSelector(availableProjectsSelector);
   const currentOrganization = useSelector(urlOrganizationSlugSelector);
   const projectSlug = useSelector(urlProjectSlugSelector);
+  const [valueSearch, setValueSearch] = useState('');
   const maxHeightPopover = window.innerHeight - MARGIN_TOP_AND_MARGIN_BOTTOM;
 
+  const filteredProjects = useMemo(
+    () =>
+      availableProjects.reduce((acc, { organizationSlug, organizationName, projects }) => {
+        const isOrganization = organizationName.toLowerCase().includes(valueSearch.toLowerCase());
+        const searchProjects = projects.filter(({ projectName }) =>
+          projectName.toLowerCase().includes(valueSearch.toLowerCase()),
+        );
+
+        return isOrganization || searchProjects.length > 0
+          ? [
+              ...acc,
+              {
+                organizationSlug,
+                organizationName,
+                projects: searchProjects,
+              },
+            ]
+          : acc;
+      }, []),
+    [valueSearch, availableProjects],
+  );
+
   const onClose = () => {
     closeSidebar();
     closePopover();
   };
 
+  const handleChange = (event) => {
+    setValueSearch(event.target.value);
+  };
+
+  const handleClear = () => {
+    setValueSearch('');
+  };
+
   return (
     <div className={cx('organizations-popover')}>
+      <div className={cx('organizations-search')}>
+        <ThemeProvider theme={variant}>
```

Comment:
I suppose it will be always `dark` in the sidebar. Can we just use the `theme="dark"` here?